### PR TITLE
Bugfix/radio buttons new schema

### PR DESCRIPTION
**Pull request by manuel-jenny**

### CODE REVIEW

File: src/components/segmented-radio-buttons/SegmentedRadioButtons.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  constructor (props) {
+    super(props)
+    const { input, items } = props
+
+    this.handleOnClick = this.handleOnClick.bind(this)
+
+    const selectedOption = items.filter((item) => item.properties.preSelected === true)
+    const valueOfSelected = selectedOption[0] ? selectedOption[0].properties.value : null
+
+    console.log('segRadio selectedOption', selectedOption)
```
Comment (AndyOGo) [inline src/components/segmented-radio-buttons/SegmentedRadioButtons.js]: @manuel-jenny 
Logs würd ich noch entfernen

File: src/components/segmented-radio-buttons/SegmentedRadioButtons.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { Field } from 'redux-form'
+
+import SegmentedControlItem from './segmented-control/SegmentedControlItem'
```
Comment (AndyOGo) [inline src/components/segmented-radio-buttons/SegmentedRadioButtons.js]: @manuel-jenny 
@lina-salih 
Is this supposed to be renamed to?

File: src/components/segmented-radio-buttons/SegmentedRadioButtons.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      input: { name },
+      orientation,
+      items,
+      meta,
+      label,
+      events,
+      ...props
+    } = this.props
+
+    const radioButtonsClasses = classNames('m-segmented-control', {
```
Comment (AndyOGo) [inline src/components/segmented-radio-buttons/SegmentedRadioButtons.js]: @lina-salih 
@manuel-jenny 
The class names to not match with the component name

File: src/components/bullet-radio-buttons/bullet-radio-buttons.scss [hunk 4]
```diff
@@ -1,15 +1,15 @@
 @import '../../../node_modules/@axa-ch/patterns-library/src/styles/mixins/typo';
 @import '../../../node_modules/@axa-ch/patterns-library/src/styles/settings/colors';
 
-.m-radio-buttons {
+.m-bullet-radio-buttons {
```
Comment (AndyOGo) [inline src/components/bullet-radio-buttons/bullet-radio-buttons.scss]: 👍

File: src/components/segmented-radio-buttons/segmented-radio-buttons.scss [hunk 5]
```diff
@@ -1,11 +1,11 @@
-@import '../../../../node_modules/@axa-ch/patterns-library/src/styles/mixins/typo';
-@import '../../../../node_modules/@axa-ch/patterns-library/src/styles/settings/colors';
+@import '../../../node_modules/@axa-ch/patterns-library/src/styles/mixins/typo';
+@import '../../../node_modules/@axa-ch/patterns-library/src/styles/settings/colors';
 
-.m-segmented-control {
+.m-segmented-radio-buttons {
```
Comment (AndyOGo) [inline src/components/segmented-radio-buttons/segmented-radio-buttons.scss]: @manuel-jenny 
Is this for one item?
If so I would use singular class names, not plural.

Comment (manuel-jenny) [reply src/components/segmented-radio-buttons/segmented-radio-buttons.scss]: @AndyOGo 
Well the question is mainly if we call the whole thing a radio button with multiple radio button items.
In that case we can change it to be singular. The same change would apply to the bullet radio button(s) as well.

What do you think?

Comment (AndyOGo) [reply src/components/segmented-radio-buttons/segmented-radio-buttons.scss]: I see, then it's fine for me this way.

@LucaMele What do you think?

**Pull request merged**